doc/40_observatory/OBS_Augment_table_with_census.md

```diff
@@ -0,0 +1,125 @@
+
+## Name
+
+OBS_AUGMENT_TABLE_WITH_CENSUS - adds the requested dimension to the input table aggregating appropriately depending on the tables
+geometry type.
+
+## Synopsis
+
+```postgresql
+OBS_AUGMENT_TABLE_WITH_CENSUS (
```

Comment:
ALL CAPITALS IS BAD if for only for readability.

Let's stick to `OBS_FunctionName`